hubspot3/crm_associations.py

```diff
@@ -179,3 +182,62 @@ def link_owner_to_company(self, owner_id, company_id):
             to_object=company_id,
             definition=Definitions.OWNER_TO_COMPANY,
         )
+
+    def chunks(self, it, size=100):
+        l = len(it)
+        it = iter(it)
```

Comment:
Aqui acho que podia ser outro nome além de `it`, pra não ficar sobrescrevendo a variável.